cd/publish_images.sh

```diff
@@ -98,3 +98,10 @@ pushContainer gchq/gaffer-road-traffic-loader "${GAFFER_VERSION}"
 pushContainer gchq/gaffer-pyspark-notebook "${GAFFER_VERSION}"
 pushContainer gchq/gaffer-jhub-options-server "${JHUB_OPTIONS_SERVER_VERSION}"
 pushContainer gchq/spark-py "${SPARK_VERSION}"
+
+# Also push legacy versions
+source "${ROOT_DIR}"/docker/accumulo1.env
+pushContainer gchq/hdfs "${HADOOP_VERSION}"
+pushContainer gchq/accumulo "${ACCUMULO_VERSION}"
+pushContainer gchq/gaffer "${GAFFER_VERSION}-accumulo-${ACCUMULO_VERSION}"
+pushContainer gchq/gaffer-rest "${GAFFER_VERSION}-accumulo-${ACCUMULO_VERSION}"
```

Comment:
This will cause a problem where the tags for `2`, `2.0` and `2.0.0` point to the legacy images.
This could be fixed by doing these before the current ones. A better fix would be changing the script to not tag them in this way at all.